lib/active_model/command/version.rb

```diff
@@ -1,5 +1,5 @@
 module ActiveModel
   module Command
-    VERSION = "0.1.4"
+    VERSION = "0.2.0"
```

Comment:
Do we want to call this version `1.0.0`? Seems like there are enough changes to justify a major bump.

Reply:
There are additional changes I would like to introduce that this sets the stage for before solidifying the first public release. Additionally `prepend` is still supported (with deprecation warnings) and there are no breaking changes, so I think this falls safely under a minor bump either way.